CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ accidentally triggering the load of a previous DB version.**
 * #3899 Fix segfault in Continuous Aggregates
 * #4225 Fix TRUNCATE error as non-owner on hypertable
 * #4259 Fix logic bug in extension update script
+* #4236 Fix potential wrong order of results for compressed hypertable with a non-default collation
```

Comment:
Can we add this as text below unreleased:
```
If you use compression with a non-default collation on a segmentby-column you might have to recompress the affected hypertable.
```

Reply:
done